runtime/stackstorm-1ppc/README.md

```diff
@@ -102,16 +86,9 @@ See [official document](https://docs.stackstorm.com/reference/ha.html#sharing-co
     - st2actionrunner
     - st2sensorcontainer
 
-## For developers
-
-### Policies
-
-- Use `stackstorm:stackstorm` as a base image to avoid code duplication
-- Try to make changes minimal as possible so that it can be easily integrated into one *universal* image in the future
-
 ### Notes
 
-- In `/etc/nginx/nginx.conf` some hostnames are hardcoded and cannot set via environment variables
+- In `/etc/nginx/nginx.conf` some hostnames are hardcoded and cannot set via environment variables, which means those services must be accessible by that hardcoded hostnames (and are configured so by default)
```

Comment:
Just some grammar improvements:

> ...some hostnames are hardcoded and cannot be set via environment variables. This means those services must be accessible by those hardcoded hostnames

Also, I'm not sure what is meant by "(and are configured so by default)"

Reply:
Also, don't you mean `/etc/nginx/conf.d/st2.conf`?

Reply:
I've fixed this up so `st2-1ppc.conf` is written into the image. The appropriate entrypoint script creates a symlink from `st2.conf` -> `st2-base.conf` or `st2-1ppc.conf` as required.

Reply:
> I'm not sure what is meant by "(and are configured so by default)"

I meant, container names defined in example `docker-compose.yml` under `/runtime/compose-1ppc` is set appropriately.